client/components/text-diff/README.md

```diff
@@ -0,0 +1,39 @@
+# TextDiff
+
+<img width="183" alt="textdiff-example" src="https://user-images.githubusercontent.com/4335450/33998736-5a0a274c-e0e0-11e7-97a2-bb0b8cc4c77b.png">
+
+TextDiff is a component useful for visualizing changes (diffs) to bodies of text.
+
+## Usage
+
+TextDiff should be passed an array of 'changes'. These changes might be additions, deletions or copies.
+In the following example the changes array represents a title rename by including the deletion of the old title and addition of a new title:
+
+```jsx
+import TextDiff from 'components/text-diff';
+
+export default function RenamedTitle() {
+	const changes = [ {
+		op: 'del',
+		value: 'Old Title.',
+	}, {
+		op: 'add',
+		value: 'New Title!',
+	} ];
+
+	return (
+		<h2>
+			<TextDiff changes={ changes } />
+		</h2>
+	);
+}
+```
+
+## Props
+
+The following props are available to customize the accordion:
+
+- `changes`: An array of objects representing a content change, of shape:
```

Comment:
What do you think about renaming this to `operations`? `copy` isn't so much a change.

Reply:
Sounds good to me! I think it'd mean we need a bit of a re-work of the would-be `operation.op` property. I was thinking `type` (as in `changes.type`) would be nice anyway, even more fitting with `operations.type`.

Reply:
Good thought.  `operations.type` makes good semantic sense to me.

Reply:
I've updated all instances of `changes` to `operations` but left `op` as-is. It's not quite ideal in terms of semantics but avoids having to amend the data returned from the API.